zerver/models.py

```diff
@@ -2640,33 +2640,29 @@ class UserTopic(models.Model):
         default=datetime.datetime(2020, 1, 1, 0, 0, tzinfo=datetime.timezone.utc)
     )
 
-    # Implicitly, if a UserTopic does not exist, the (user, topic)
-    # pair should have normal behavior for that (user, stream) pair.
+    class VisibilityPolicy(models.IntegerChoices):
+        # A normal muted topic. No notifications and unreads hidden.
+        MUTED = 1, _("Muted topic")
```

Comment:
How are the strings here used? I'm not sure they are displayed in the UI/API and thus whether they should be tagged for translation; if they are tagged for translation, I expect they'll need to be tagged using `gettext_lazy` so that they are translated at time of use.

Reply:
Ah, yes. These will appear neither in the UI nor in the API. I'll remove the translation tags.